coding-intro/coding-intro-core/core-concepts/programs-and-data.md

```diff
@@ -56,9 +57,13 @@ That's why we group our data into distinct **data types**, the three main ones b
 | Number  | 5,-0.5     |
 | Boolean | True, False |
 
-What would we get if we tried to take the square root of `"cat"`?
+These data types save us from attempting number-related operations on texts and vice versa.
+
+> <emoji> <question>
+>
+> Leave a comment or view some of the other comments for inspiration before moving on.
```

Comment:
What's the emoji and question here?

I left <emoji> and <question> as placeholders in my comment 😅 

Based on your latest changes from above, we can put back the square root of "cat" again:

```suggestion
> 🐈 Can a computer tell us what's the square root of "cat"?
>
> Leave a comment or view some of the other comments for inspiration before moving on.
```